src/lib/queries/lexical-query.js

```diff
@@ -168,6 +168,8 @@ export default class LexicalQuery extends Query {
       }
     }
 
+    LexicalQuery.evt.HOMONYM_READY.pub(this.homonym)
```

Comment:
I think this was after the retrieval of the lexicon options because otherwise the user will see short definitions change if the adapter also inserted short definitions.

Reply:
oh, yes, sorry - I missed it while doing investigation! Will fix!

Reply:
@irina060981 I think this still needs to be put back?

Reply:
I have checked - somehow I didn't commit this fix - will fix it right now, I am sorry!